common/desktop-exports

```diff
@@ -2,6 +2,13 @@
 # Launcher common exports for any desktop app #
 ###############################################
 
+# On Fedora $SNAP is under /var and there is some magic to map it to /snap.
+# We need to handle that case and reset $SNAP
+if [[ $SNAP == /var/lib/snapd/* ]];
+then
+  export SNAP=`echo $SNAP | sed -e "s/\/var\/lib\/snapd//g"`
```

Comment:
I think this could be simplified in:
`SNAP=`echo $SNAP | sed -e "s#/var/lib/snapd##g"``

1. No need for export, $SNAP was already exported.
2. Using # as a separator